src/views/HomeView.vue

```diff
@@ -196,4 +241,17 @@ h1 {
 #fontsize-select {
   margin-right: 10px;
 }
+
+.grid {
+  align-self: center;
+  justify-content: space-around;
+  flex-wrap: wrap;
+  gap: 1rem;
+}
+
```

Comment:
Ich würde eher nicht noch weitere css regeln zu klassen aus pico css hinzufügen, weil man sich sonst in anderen komponenten wundert.